source/includes/harvest/_users.md

```diff
@@ -353,3 +353,85 @@ email | Yes | string | The user's email address. Must be a valid email address.
 send_email_invite* | No | boolean | If true, an email will be sent to the above email address inviting them to login. If false, nothing happens. Default is false.
 
 \* - A newly created user will not be able to login until they create a password via the invitation link or configured in an SSO system.
+
+## PUT: Add a Job Permission
+
+```shell
+curl -X PUT 'https://harvest.greenhouse.io/v1/users/{id}/permissions/jobs'
+-d '{ "job_id": {job_id}, "user_role_id": {user_role_id} }'
+-H "On-Behalf-Of: {greenhouse user ID}"
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+> The above command takes a JSON request, structured like this:
+
+```
+{
+  "id": 271232,
```

Comment:
Will the user ever specify the ID here? Won't they just choose the job_id and user_role_id?